pkg/kubectl/cmd/cmd.go

```diff
@@ -331,7 +332,16 @@ func DefaultClientConfig(flags *pflag.FlagSet) clientcmd.ClientConfig {
 
 func checkErr(err error) {
```

Comment:
Its already covered in https://github.com/GoogleCloudPlatform/kubernetes/issues/2146

----- Original Message -----
> > @@ -331,7 +332,16 @@ func DefaultClientConfig(flags *pflag.FlagSet)
> > clientcmd.ClientConfig {
> >  
> >  func checkErr(err error) {
> 
> sgtm. The scope of this PR is addressing the sever-vs-client error origin
> (#3937). Are you ok with merging this as is and opening a separate issue for
> the (mostly mindless) task of converting all commands to use the above error
> handling pattern?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/5016/files#r25819463